nusight2/src/client/components/localisation/network.ts

```diff
@@ -41,6 +42,12 @@ export class LocalisationNetwork {
     robot.fieldLinesDots.rPWw = fieldLines.rPWw.map((rPWw) => Vector3.from(rPWw).applyMatrix4(robot.Hfw));
```

Comment:
```suggestion
    robot.fieldLinesDots.rPFf = fieldLines.rPWw.map((rPWw) => Vector3.from(rPWw).applyMatrix4(robot.Hfw));
```

Aside... shouldn't this be named `rPFf`?

Reply:
The network layer shouldn't be converting to field space, that should happen constantly so that if robot.Hfw updates (localisation updates) then it's recalculated

Reply:
Where abouts would we move the transform to have it updated online with a localisation update?

Reply:
Correct, it moves into the model as a computed property, similar to how `Hft` is defined in `robot_model.ts`:

```ts
export class LocalisationRobotModel {

  // ...

  /** Ball position in field space */
  @computed
  get rBFf(): Vector3 {
    return this.ball.rBWw.applyMatrix4(robot.Hfw);
  }
}
```

Now you can use `robot.rBFf.toArray()` in the view.